packages/wallet-core/src/tests/direct-funder.test.ts

```diff
@@ -388,17 +394,19 @@ test('pure objective cranker start to finish', () => {
   See https://github.com/statechannels/statechannels/blob/2b188f919cc0d1b3d4a12aca8918ddc0fb1bbaca/packages/server-wallet/src/__test-with-peers__/create-and-close-channel/direct-funding.test.ts#L73
 
   For the purpose of this test, I am arbitrarily deciding that the actions & events occur in this order:
-  1. Alice signs the preFS, triggers preFS action 1
-  2. Bob receives preFS event 1, trigers preFS action 2
-  3. Alice receives preFS event 2, triggers deposit action 1
-  4. Alice receives deposit event 1, does nothing
-  5. Bob receives deposit event 1, triggers deposit action 1
-  6. Bob receives deposit action 2 (UNFINALIZED)
-  7. Alice receives deposit action 2 (UNFINALIZED)
-  8. Bob receives deposit action 2 (FINALIZED), triggers postFS action 1
-  9. Alice receives deposit action 2 (FINALIZED), triggers postFS action 2
-  10. Alice receives postFS event 1, is finished
-  11. Bob receives postFS event 1, is finished
+  1. Alice tries to crank a non-appproved objective. No actions are taken
+  2. Alice approveves objective, signs the preFS, triggers preFS action 1
+  3. Bob receives preFS event 1, and tries to crank a non-approved objective
+  4. Bob approves objective, signs preFS, trigers preFS action 2
+  5. Alice receives preFS event 2, triggers deposit action 1
+  6. Alice receives deposit event 1, does nothing
+  7. Bob receives deposit event 1, triggers deposit action 1
+  8. Bob receives deposit action 2 (UNFINALIZED)
+  9. Alice receives deposit action 2 (UNFINALIZED)
+  10. Bob receives deposit action 2 (FINALIZED), triggers postFS action 1
+  11. Alice receives deposit action 2 (FINALIZED), triggers postFS action 2
+  12. Alice receives postFS event 1, is finished
+  13. Bob receives postFS event 1, is finished
```

Comment:
This looks like a maintenance headache 😆

Reply:
It is, but it is useful for now.